configs/openmrs/initializer_config/dispositions/dispositionConfig.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
I think this is being brought in by the Ref App distro so that it doesn't have to apart of this distro.